src/Fragments/InputFragment.tsx

```diff
@@ -46,6 +47,20 @@ const InputElement = styled.input<InputFragmentProps>`
         opacity: 0.6;
     }
 
+    ${({width}):string=>{
+        if(width){
+            switch (typeof(width)){
+            case 'number':
+                return `width:${width}px;`
+            case 'string':
+                return `width:${width};`
+            default:
+                return ''
+            }
+        }
+        return ''
+    }}
+
```

Comment:
Why is this needed?

Reply:
because if I do not this Input was not responding to setting of width prop

<Input ... width={65} /> or <Input ... width='65px' /> did not work if I do not do this.

Reply:
What do you mean, the width can be modified before no?